FuturePortal/RickvanderStaaij
# Add Travis, Deployer and TravisDeployer

### What

Migrate form PHPCI and Rocketeer to TravisCI and Deployer

### How to test

1. See that the deployment config is correct
1. See that the TravisCI build succeeds
